rdt/performance/datasets/text.py

```diff
@@ -1,4 +1,4 @@
-"""Dataset Generators for Text transformers."""
+"""Dataset Generators for 'text' transformers."""
```

Comment:
Should we just delete this file now since we have the id one testing the exact same thing?

Reply:
I left it so we can test the `text` input still works as sdtype.